packages/contracts-bedrock/deploy/001-AddressManager.ts

```diff
@@ -11,8 +15,15 @@ const deployFn: DeployFunction = async (hre) => {
     contract: 'AddressManager',
     args: [],
     postDeployAction: async (contract) => {
-      // Owner is temporarily set to the deployer.
       await assertContractVariable(contract, 'owner', deployer)
+      await doOwnershipTransfer({
+        false,
```

Comment:
i'm confused about the `false` here, we want `false: false` in the object?

Reply:
Oops, good catch! That was the `isLiveDeployer` that i accidentally removed